developerportal/settings/base.py

```diff
@@ -322,3 +322,16 @@
 CELERY_ACCEPT_CONTENT = ["application/json"]
 CELERY_TASK_SERIALIZER = "json"
 CELERY_RESULT_SERIALIZER = "json"
+CELERY_TIMEZONE = "UTC"
+CELERY_ENABLE_UTC = True
+
+_redis_cache_url = os.environ.get("REDIS_URL", "redis://redis:6379/0")
+_redis_cache_url = _redis_cache_url.replace("/0", "/1")
```

Comment:
@limed let me know if this assumption of DB 0 will be a problem. Indeed, does the (secret) value for `REDIS_URL` in k8s end with a DB number? if not i will amend this code to handle that, too

Reply:
No the secret just uses the redis url and no `/0`

Reply:
OK - will amend my PR to handle that. Thanks dude